### DISCUSSION TITLE
Stella - My Xbox controller is bluetooth connected to the Mac, and is there a way to use it rather than the keyboard?

### DISCUSSION THREAD

**Discussion by supremeai** [stella-emu/stella#999] [Q&A]
Stella - My Xbox controller is bluetooth connected to the Mac, and is there a way to use it rather than the keyboard?

**Comment by thrust26** [comment]
You can use your controllers to navigate the UI. Probably to have to adapt the button/direction UI mappings so that they fit to your controller.

**Reply by supremeai** [reply]
To clarify, do you mean adapt the button/direction UI mappings in Stella?
